docs/source/release/v3.14.0/reflectometry.rst

```diff
@@ -65,6 +65,8 @@ Improved
 ########
 
 - :ref:`algm-ReflectometryReductionOneAuto` now supports the Wildes method for polarization corrections as well as Fredrikze when configured in the parameters file.
+- :ref:`algm-ReflectometryReductionOneAuto` now uses spectrum numbers for it's processing instructions instead of workspace indecies
```

Comment:
- We need to mention `CTW` and `CTWA` too. I think one line in the notes which mentions all 4 is fine.